oneview_redfish_toolkit/api/storage_collection.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import collections
+from oneview_redfish_toolkit.api.redfish_json_validator \
+    import RedfishJsonValidator
+
+
+class StorageCollection(RedfishJsonValidator):
+    """Creates a Storage Collection Redfish dict
+
+        Populates self.redfish with some hardcoded StorageCollection
+        values and with the...
+    """
+
+    SCHEMA_NAME = 'StorageCollection'
+
+    def __init__(self, uuid):
+        """StorageCollection constructor
+
+            Populates self.redfish with a hardcoded StorageCollection
```

Comment:
Are we not gonna consider the case of multiple storage in a server-hardware?

Reply:
The OneView SHT storageCapabilities is an object and not an array so we are going to model this as a collection with a single member.